src/routes/_public/report/-components/summary-tab.tsx

```
@@ -89,6 +91,39 @@ export function ResultsTable({ categories, calculatedReports }: ResultsTableProp
+                <div className="flex flex-col gap-1">
+                  <div className="font-light text-secondary-content">Your Blindspot</div>
+                  <div className="text-lg font-bold text-primary-content">
+                    {calculatedReport[0].traitName}
+                  </div>
+                </div>
+                <Separator />
+                <div className="flex flex-col gap-1">
+                  <div className="font-light text-secondary-content">Summary</div>
+                  <div
+                    className="font-light text-primary-content"
+                    dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)